gpt_conf.py

```diff
@@ -33,6 +33,13 @@ class GPTConfig:
     quantize_mlp_up: bool = False
     quantize_mlp_down: bool = False
 
+    binarize_q: bool = False
```

Comment:
I think we want to rename these to binarize_c_attn_q, c_attn_k, ...

Reply:
Actually, I think we can get rid of these options, and just use quantize_c_attn. But, add binarize_linear and ternarize_linear to the "linear_variant" argument